src/main/java/io/vertx/kafka/admin/AclBinding.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.vertx.kafka.admin;
+
+import java.util.Objects;
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
```

Comment:
blank line

Reply:
@vietj as the Vert.x expert, what's your opinion on this? how the other libraries are behaving?

Reply:
@vietj  Can we get your input on this to clarify what is desired in this project? Thanks!

Reply:
you should look at `@GenIgnore(GenIgnore.PERMITTED_TYPES)` which allow to use non vertx types